### ISSUE TITLE
Contacting Ubudu

### ISSUE THREAD

**Issue by brandecho** [Ubudu/Cordova-SDK#1]
I have sent several emails and submitted online forms multiple times on the Ubudu site and have yet to receive a response from anyone at Ubudu. We are looking to purchase several Ubudu beacon kits but prior to doing so, we would like to understand the setup for creating triggers from the user app and then notify and request assist from the store manager as seen on the following video on Youtube at marker 1min 15sec:  https://youtu.be/Q0H0fV5TCQI 

Apologizing in advance for having to post an issue on this Github page but Im left with no choice and hope someone will contact me to provide more insight prior to purchasing approximately 10 or more beacon kits.

**Comment by ghost** [comment]
> On 17 Feb 2018, at 13:01, brandecho <notifications@github.com> wrote:
> 
> I have sent several emails and submitted online forms multiple times on the Ubudu site and have yet to receive a response from anyone at Ubudu. We are looking to purchase several Ubudu beacon kits but prior to doing so, we would like to understand the setup for creating triggers from the user app and then notify and request assist from the store manager as seen on the following video on Youtube at marker 1min 15sec: https://youtu.be/Q0H0fV5TCQI <https://youtu.be/Q0H0fV5TCQI>
> Apologizing in advance for having to post an issue on this Github page but Im left with no choice and hope someone will contact me to provide more insight prior to purchasing approximately 10 or more beacon kits.
> 

Have you tried the contact information at: http://ubudu.com/contact.html <http://ubudu.com/contact.html> ?
(phone and email?)

I’ve not had contact with ubudu since 2 years, and from this page, I gather they moved to new offices.

-- 
__Pascal J. Bourguignon__

**Comment by brandecho** [comment]
Yes I have. Submitted a few times. Definitely a concern as we dont want to integrate into our cordova apps if no support is available; not to mention, purchase beacons.

**Comment by brandecho** [comment]
Looks like they are updating the blog. Last post was in Jan 2018. Maybe I try to send a tweet and hopefully get a response.

**Comment by MGasztold** [comment]
Hello,

I am sorry there was no response for so long. Unfortunately due to limited team bandwidth and lots of projects we are not currently supporting or developing Cordova SDK. 

We recommend you to use our Android or iOS mobile SDK which are actively developed.
